lib/blank-node.js

```diff
@@ -1,12 +1,16 @@
-function BlankNode (id) {
-  this.value = id || ('b' + (++BlankNode.nextId))
-}
+class BlankNode {
+  constructor (id) {
+    this.value = id || ('b' + (++BlankNode.nextId))
+  }
 
-BlankNode.prototype.equals = function (other) {
-  return !!other && other.termType === this.termType && other.value === this.value
-}
+  equals (other) {
+    return !!other && other.termType === this.termType && other.value === this.value
+  }
 
-BlankNode.prototype.termType = 'BlankNode'
```

Comment:
I think for performance reasons we should keep the old code for the `termType` string value to avoid the overhead of a function call.